src/XmlTransformer.cs

```diff
@@ -109,9 +109,6 @@ static void Main(string[] args)
 
                 if (!options.MultipleOutput)
                 {
-                    if ( options.Html && !options.NoHeader )
```

Comment:
It turns out this is incorrect. It leads to multiple  trailers when all the transforms are going to the same file. It was the first instance of WriteHtmlTrailer that needed to be suppressed in this case.

I'm going to restructure the code to take care of this under issue #6. I think there should just be two different methods for doing output: one for merging to a single file and another to doing one output per input.